package.json

```diff
@@ -43,5 +46,10 @@
     "url": "https://github.com/BrianMacIntosh/alexa-skill-test-framework/issues",
     "email": "brianamacintosh@gmail.com"
   },
-  "repository": "BrianMacIntosh/alexa-skill-test-framework"
+  "repository": "BrianMacIntosh/alexa-skill-test-framework",
+  "devDependencies": {
+    "alexa-sdk": "^1.0.10",
```

Comment:
No, i didn't realized that, cause i've been using the package without the alexa sdk. But, yes, now that you say it's required for the tests i totally get it